changes/5521.dependencies

```diff
@@ -0,0 +1 @@
+Updated most dependencies to the latest versions available (as of 2024-04-01) that could be supported without significant code changes in Nautobot.
```

Comment:
Should we list the dependencies explicitly here?

Reply:
We could, I'm just not certain that that's interesting/useful information to most users. Open to persuasion on that point though.

Reply:
```suggestion
Updated most dependencies to the latest versions available as of 2024-04-01.
```